core/src/toga/widgets/webview.py

```diff
@@ -135,6 +139,22 @@ def set_content(self, root_url: str, content: str) -> None:
         """
         self._impl.set_content(root_url, content)
 
+    def get_cookies(
```

Comment:
Pythonic APIs tend not to have getters and setters; `set_content` is a special case because we actually need to pass in multiple arguments to the set, but other Toga APIs lean towards properties (c.f. `user_agent`).